test/skip-subtypes/generated/BodyComplex/README.md

```diff
@@ -1,88 +1,83 @@
+- `@azure/identity` that contains different credentials for you to authenticate the client using Azure Active Directory.
+Install both packages using the below commands.
+Alternatively, you can add these to the dependencies section in your package.json and then run `npm install`.
 ```bash
 npm install
+npm install @azure/identity
 ```
+Please note that while the credentials from the older `@azure/ms-rest-nodeauth` and `@azure/ms-rest-browserauth` packages are still supported, these packages are in maintenance mode receiving critical bug fixes, but no new features.
+We strongly encourage you to use the credentials from `@azure/identity` where the latest versions of Azure Active Directory and MSAL APIs are used and more authentication options are provided.
 
-### How to use
+### How to Use
 
-#### nodejs - client creation and getValid basic as an example written in TypeScript.
-
-##### Install @azure/ms-rest-nodeauth
-
-- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
-```bash
-npm install @azure/ms-rest-nodeauth@"^3.0.0"
-```
+There are multiple credentials available in the `@azure/identity` package to suit your different needs.
+Read about them in detail in [readme for @azure/identity package](https://www.npmjs.com/package/@azure/identity).
+To get started you can use the [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/README.md#defaultazurecredential) which tries different credentials internally until one of them succeeds.
+Most of the credentials would require you to register to [create an Azure App Registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals#application-registration) first.
+#### nodejs - client creation and getValid basic as an example written in JavaScript.
 
 ##### Sample code
 
-While the below sample uses the interactive login, other authentication options can be found in the [README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package
 ```typescript
-const msRestNodeAuth = require("@azure/ms-rest-nodeauth");
+const { DefaultAzureCredential } = require("@azure/identity");
 const { AutoRestComplexTestService } = require("");
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new AutoRestComplexTestService(creds, subscriptionId);
-  client.basic.getValid().then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
+const creds = new DefaultAzureCredential();
+const client = new AutoRestComplexTestService(creds, subscriptionId);
+client.basic.getValid().then((result) => {
+  console.log("The result is:");
+  console.log(result);
 }).catch((err) => {
+  console.log("An error occurred:");
   console.error(err);
 });
 ```
 
 #### browser - Authentication, client creation and getValid basic as an example written in JavaScript.

```

Comment:
In the header for node, we don't have `Authentication`. We should add it there

Reply:
Done.